Dockerfile

```diff
@@ -5,4 +5,4 @@ WORKDIR /code
 COPY collector/target/collector-0.4.0-cognite-3.jar /code/stackdriver-zipkin.jar
 
 EXPOSE 9411 9100
-ENTRYPOINT [ "sh", "-c", "java -Djava.security.egd=file:/dev/./urandom -jar /code/stackdriver-zipkin.jar" ]
+ENTRYPOINT [ "sh", "-c", "java -XX:+CrashOnOutOfMemoryError -Djava.security.egd=file:/dev/./urandom -jar /code/stackdriver-zipkin.jar" ]
```

Comment:
If you haven't already, could you test this in the infrastructure cluster and report the results? Set the memory request/limit values to something fairly low. The existing `infrastructure-hello` deployment should be sending traces, so you can hit that a few times and make sure traces are sent.

Check if the `kube_pod_container_status_restarts` metric increases as expected.

Reply:
Will do.